java/test/com/liferay/nativity/test/TestDriver.java

```diff
@@ -232,9 +232,9 @@ private static void _updateFileIcon(FileIconControl fileIconControl) {
 	private static String _testFile =
 		"C:/Users/liferay/Documents/liferay-sync/Sync.pptx";
 	private static String _testFolder =
-		"C:/Users/liferay/Documents/liferay-sync/My Documents (test)/temp";
+		"C:\\Users\\sync\\Desktop\\Sarat";
```

Comment:
Well, forward slashes don't work for windows, made everything backslashes. Also, made it cross-user: "C:\\Liferaytest\\"